src/backend/api/backend.did

```diff
@@ -247,6 +247,56 @@ type UpdateProposalReviewResponse = variant {
   err : Err;
 };
 
+type ReviewCommitState = variant {
+  reviewed : record {
+    matches_description : bool;
+    comment : opt text;
+    highlights : vec text;
+  };
+  not_reviewed;
+};
+
+type ProposalReviewCommit = record {
+  proposal_review_id : text;
+  user_id : text;
+  created_at : text;
+  commit_sha : text;
+  state : ReviewCommitState;
+};
+
+type CreateProposalReviewCommitRequest = record {
+  proposal_review_id : text;
+  commit_sha : text;
+  state : ReviewCommitState;
+};
+
+type CreateProposalReviewCommitResponse = variant {
+  ok : record {
+    id : text;
+    proposal_review_commit : ProposalReviewCommit;
+  };
+  err : Err;
+};
+
+type UpdateProposalReviewCommitRequest = record {
+  id : text;
+  state : ReviewCommitState;
```

Comment:
Similar as above for updating.

Reply:
In this update request, the `state` field is the only one that contains information to update. Is there a case in which an empty (only with`id`) update request is sent to the api?